test/internet/test-dns-ipv6.js

```diff
@@ -139,21 +144,22 @@ TEST(function test_lookup_ipv6_hint(done) {
 });
 
 TEST(function test_lookup_ip_ipv6(done) {
-  const req = dns.lookup('::1',
-                         common.mustCall((err, ip, family) => {
-                           assert.ifError(err);
-                           assert.ok(isIPv6(ip));
-                           assert.strictEqual(family, 6);
+  const req = dns.lookup(
+    '::1',
+    common.mustCall((err, ip, family) => {
+      assert.ifError(err);
+      assert.ok(isIPv6(ip));
+      assert.strictEqual(family, 6);
 
```

Comment:
How about defining a constant for `::1` as well?

Reply:
`localhost` is defined for IPv4 and IPv6:

https://github.com/nodejs/node/blob/27e12e75244343ffcde4b899a31e88d452fa4d6e/test/common/index.js#L183-L185